Dockerfile

```diff
@@ -1,31 +1,27 @@
-FROM ubuntu:16.04
+# To run: docker run -v /path/to/wsgi.py:/var/www/indexd/wsgi.py --name=indexd -p 81:80 indexd
+# To check running container: docker exec -it indexd /bin/bash 
+
+FROM quay.io/cdis/py27base:pybase2-1.0.1
+
 MAINTAINER CDIS <cdissupport@opensciencedatacloud.org>
 
-RUN apt-get update && apt-get install -y sudo python-pip git python-dev libpq-dev apache2 libapache2-mod-wsgi curl vim libssl-dev libffi-dev \ 
- && apt-get clean && apt-get autoremove \
- && rm -rf /var/lib/apt/lists/*
+RUN mkdir /var/www/indexd \
+	&& chown www-data /var/www/indexd
+
 COPY . /indexd
+COPY ./deployment/uwsgi/uwsgi.ini /etc/uwsgi/uwsgi.ini
+
 WORKDIR /indexd
+
+RUN python -m pip install -r requirements.txt
 RUN COMMIT=`git rev-parse HEAD` && echo "COMMIT=\"${COMMIT}\"" >indexd/index/version_data.py
 RUN VERSION=`git describe --always --tags` && echo "VERSION=\"${VERSION}\"" >>indexd/index/version_data.py
 RUN python setup.py install
 
-RUN mkdir -p /var/www/indexd/ && chmod 777 /var/www/indexd && cp /indexd/wsgi.py /var/www/indexd/wsgi.py && cp /indexd/bin/indexd /var/www/indexd/indexd
-
-#
-# Custom apache2 logging - see http://www.loadbalancer.org/blog/apache-and-x-forwarded-for-headers/
-#
-COPY ./deployment/apache-indexd.conf /etc/apache2/sites-available/apache-indexd.conf
```

Comment:
Remove `deployment/apache-indexd.conf` too perhaps?

Reply:
Looking into that file, it seems we missed out [the Origin Access Control header](https://github.com/uc-cdis/indexd/commit/b5b3114a1fabcbcef848f823bbf1cec45f66bcfd#diff-3254677a7917c6c01f55212f86c57fbfR17) in the new image - just a reminder here, we probably added that in the `revproxy` service already.

Reply:
I removed this file